server/src/test/java/org/apache/druid/server/http/CoordinatorDynamicConfigTest.java

```diff
@@ -201,7 +197,7 @@ public void testSerde() throws Exception
         1,
         2,
         whitelist,
-        1.0,
+        0.1,
```

Comment:
This value change from 1.0 to 0.1 is the default change of `killTaskSlotRatio` made in this patch.
The test should just use a builder to make it clear, or the assertion should happen in the tests, but that cleanup is for another day.